<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProject</class>
 <widget class="QMainWindow" name="CreateProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eve Project Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Project Location</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_setFolder">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Set Location</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Project Name</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lin_name">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Project Root Folder</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lab_path">
         <property name="text">
          <string>C:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Houdini build</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lin_options"/>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chb_example">
         <property name="text">
          <string>Copy Example Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_create">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Create New Project</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_docs"/>
    <addaction name="act_help"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_docs">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="act_help">
   <property name="text">
    <string>Project Creator Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
